genesyscloud/tfexporter/resource_genesyscloud_tf_export.go

```diff
@@ -44,7 +44,27 @@ func ResourceTfExport() *schema.Resource {
 				ForceNew:    true,
 			},
 			"resource_types": {
-				Description: "Resource types to export, e.g. 'genesyscloud_user'. Defaults to all exportable types.",
+				Description: "Resource types to export, e.g. 'genesyscloud_user'. Defaults to all exportable types. NOTE: This field is deprecated and will be removed in future release.  Please use the include_filter_resources or exclude_filter_resources attribute.",
+				Type:        schema.TypeList,
```

Comment:
i seen this example here - [https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations](https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations) of how you can add a deprecation warning and specify which field conflict with eachother

```golang
            "existing_attribute": {
                Type:          schema.TypeString,
                Optional:      true,
                ConflictsWith: []string{"new_attribute"},
                Deprecated:    "use new_attribute instead",
            },
            "new_attribute": {
                Type:          schema.TypeString,
                Optional:      true,
                ConflictsWith: []string{"existing_attribute"},
            },
```